packages/utils/src/loader/cdn-loader.ts

```diff
@@ -49,39 +49,41 @@ export class CDNLoader {
    * @param href The location of the SVG to load
    * @returns Promise of the SVG body
    */
-  private async loadContent (href: string): Promise<XMLHttpRequest> {
+  private async loadContent (href: string): Promise<Response> {
     try {
-      const response = await this.xmlRequest(href);
+      const response = await this.fetchRequest(href);
       return response;
     }
     catch (e) {
       // Failed response...
       this.responseCache.delete(href);
+      if (e instanceof Response) {
+        return Promise.resolve({
+          status: e.status || 0,
+          statusText: e.statusText || 'Failed to make request'
+        } as Response);
+      }
       return Promise.resolve({
-        status: 0, response: 'Failed to make request', responseText: 'Failed to make request'
-      } as XMLHttpRequest);
+        status: 0,
+        statusText: 'Failed to make request'
+      } as Response);
     }
   }
 
   /**
-   * Load and support on IE
+   * Load by fetch api
    * @param href The source or location
-   * @returns XMLHttpRequest objects after to interact servers.
+   * @returns Response objects after to interact servers.
    */
-  private async xmlRequest (href: string): Promise<XMLHttpRequest> {
-    return new Promise((resolve, reject) => {
-      const xhr = new XMLHttpRequest();
-      xhr.open('GET', href);
-      xhr.onload = (): void => {
-        if (xhr.readyState === 4 && xhr.status >= 200 && xhr.status < 400) {
-          resolve(xhr);
-        }
-        else {
-          reject(xhr);
-        }
-      };
-      xhr.onerror = (): void => reject(xhr);
-      xhr.send();
+  private async fetchRequest (href: string): Promise<Response> {
```

Comment:
Rename it to 'fetch' might be better.

Reply:
I think we might not even need this as a separate function. This can be easily merge into `loadContent` with a single catch block.

Reply:
Agreed and updated. I merges fetchRequest function into `loadContent`.